src/main/java/core/basesyntax/exceptions/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exceptions;
+
+public class RegistrationException extends Error {

```

Comment:
Extending `Error` is not recommended for custom exceptions. Consider extending `Exception` or `RuntimeException` instead, depending on whether you want it to be a checked or unchecked exception.